cross-dom-comm/hardhat.config.js

```diff
@@ -0,0 +1,35 @@
+require("@nomiclabs/hardhat-waffle");
+
+// This is a sample Hardhat task. To learn how to create your own go to
+// https://hardhat.org/guides/create-task.html
+task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
+  const accounts = await hre.ethers.getSigners();
+
+  for (const account of accounts) {
+    console.log(account.address);
+  }
+});
+
+
+const mnemonic = ' <<< Your mnemonic goes here >>> ' 
```

Comment:
Please update this to an `.env.example` We do not want to encourage insecure setups.

Reply:
Done.